cmd/aem/config.go

```diff
@@ -50,3 +40,55 @@ func (c *CLI) configInitCmd() *cobra.Command {
 	}
 	return cmd
 }
+
+func (c *CLI) configListCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Aliases: []string{"ls", "print", "values"},
+		Short:   "Read all configuration values",
+		Run: func(cmd *cobra.Command, args []string) {
+			c.SetOutput("values", c.config.Values())
+			c.Ok("config values read")
+		},
+	}
+	return cmd
+}
+
+func (c *CLI) configReadCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "read",
+		Aliases: []string{"get", "value"},
+		Short:   "Read configuration value",
+		Run: func(cmd *cobra.Command, args []string) {
+			key, _ := cmd.Flags().GetString("key")
+			template, _ := cmd.Flags().GetString("template")
+			if key == "" && template == "" {
+				c.Fail("flag 'key' or 'template' need to be specified")
+				return
+			}
+			var (
+				value string
+				err   error
+			)
+			if key != "" {
+				value, err = tplx.RenderKey(key, c.config.Values())
+				if err != nil {
+					c.Error(fmt.Errorf("cannot read config value using key '%s': %w", key, err))
+					return
+				}
+			} else {
+				value, err = tplx.RenderString(template, c.config.Values())
+				if err != nil {
+					c.Error(fmt.Errorf("cannot read config value using template '%s': %w", template, err))
+					return
+				}
+			}
+			c.SetOutput("value", value)
+			c.Ok("config value read")
+		},
+	}
+	cmd.Flags().StringP("key", "k", "", "Value key")
+	cmd.Flags().StringP("template", "t", "", "Value template")
```

Comment:
TBH, I don't get these flags. What is template?

Reply:
template is a entire string / go tpl: https://pkg.go.dev/text/template `[[ a.b.c ]]`

key is a `a.b.c`; key is a just shorthand to skip tpl delimiters

Reply:
these flags will be probably not needed as I changed the approach.

both AEMC and Task tool will load vars from dotenv files `aem.env` (VCS-tracked) then `local.env` (VCS-ignored/tracked/depends) and ultimately from `.env` (VCS-ignored)